projects/components/src/load-async/load-async.directive.ts

```diff
@@ -49,6 +53,7 @@ export class LoadAsyncDirective implements OnChanges, OnDestroy {
       this.wrapperView = this.wrapperView || this.buildWrapperView();
       this.wrapperParamsSubject.next({
         state$: this.loadAsyncService.mapObservableState(this.data$),
+        loaderType: this.htLoadAsyncLoaderType,
```

Comment:
So basically, if htLoadAsyncLoaderType changes, then this if block will not be triggered? So, how would you handle the case when loaderType changes after init?


Why don't you send `loaderType` to `mapObservableState` as another param? then this function can add this loaderType in ```interface LoadingAsyncState {
  type: LoadAsyncStateType.Loading;
}```
Also, update L52 to also include change on LoaderType